tests/h/search/index_test.py

```diff
@@ -310,6 +312,80 @@ def test_annotation_is_marked_deleted(self, es_client, factories, index, search)
         assert result.get('_source').get('deleted') is True
 
 
+class TestBatchIndexer(object):
+    def test_it_indexes_all_annotations(self, batch_indexer, es_client, factories):
+        annotations = factories.Annotation.create_batch(3)
+        ids = [a.id for a in annotations]
+
+        batch_indexer.index()
+
+        results = es_client.conn.search()['hits']['hits']
+        assert [ann.get("_id") for ann in results].sort() == ids.sort()
```

Comment:
This assert doesn't do what you expected because the `sort()` API mutates the array and then always returns `None`. Same gotcha that Hannah [encountered here](https://github.com/hypothesis/h/pull/5081#discussion_r196680399). You can use `sorted(list)` to get a sorted copy of a list.

Reply:
Sorry @sheetaluk! My bad!